superduperdb/misc/annotations.py

```diff
@@ -0,0 +1,141 @@
+import inspect
+import sys
+import warnings
+from functools import wraps
+from typing import Optional
+
+
+def PublicAPI(*args, **kwargs):
```

Comment:
Do we need `*args, **kwargs`. Why not have named arguments?

Reply:
done